Automattic/wp-calypso
# Login: Update Two-Step Authentication page

Update the 2FA screen to use the new style illustration. Change the background of the 2FA actions to white to reduce muddiness of subtle grays and increase readability. This pull request also refactors how we handle errors to address [some issues](https://github.com/Automattic/wp-calypso/pull/14214#issuecomment-305459119) found during testing.

<img width="447" alt="screen shot 2017-06-01 at 06 57 59" src="https://cloud.githubusercontent.com/assets/448298/26676967/dd237626-4697-11e7-8dd3-0c9ed9663021.png">

#### Testing instructions

1. Apply server-side patch D5843-code
2. Run `git checkout update/login-2fa-actions` and start your server, or open a [live branch](https://calypso.live/?branch=update/login-2fa-actions)
3. Open the [`Login` page](http://calypso.localhost:3000/log-in)
4. Log in to an account that has 2FA enabled with push notification
5. Confirm that the page looks like the screen shot above

Head to D5843-code for the rest of the testing instructions.
  
#### Reviews
  

cc @danhauk
